packages/embark-ui/src/components/ContractsDeployment.js

```diff
@@ -38,12 +39,14 @@ const NoWeb3 = () => (
   </Row>
 );
 
-const LayoutContract = ({contract, children, cardTitle}) => (
+const LayoutContract = ({contract, children, title = contract.className, isLoading = false, isDeployed = false, deployedTitleSuffix, notDeployedTitleSuffix}) => (
```

Comment:
Defaulting to `false` is not super useful unless you do `=== false`, but whatever, it's no big deal